src/containers/AccountConnection.jsx

```diff
@@ -13,13 +13,14 @@ class AccountConnection extends Component {
   constructor (props, context) {
     super(props, context)
     this.store = this.context.store
+    this.state = {
+      account: this.props.existingAccount
```

Comment:
Don't we need to monitor the props to get all change from `props.existingAccount` using `componentWillReceiveProps()` lifecycle hook? Since it's only in the constructor, the state won't be updated if the props changes after this component mounting. What do you think? Or maybe it's wanted to get this props only at the beginning of the component.